events/forms.py

```diff
@@ -18,3 +18,25 @@ def __init__(self, *args, **kwargs):
         self.action = None
         super(EventForm, self).__init__(*args, **kwargs)
         self.fields['estimated_end_date'].initial = timezone.now() + relativedelta(months=1)
+
+
+class EventCreateForm(forms.ModelForm):
+    """
+    Adding new event by politikon users form
+    """
+    def save(self, commit=True):
+        """
+        Save event form to object
+        :param commit: If True, then the changes to ``instance`` will be saved to the database.
```

Comment:
it probably always should be, but every person should once per 30 sec for example (setting new db info: 'last_event_submit' date in user profile information)?